### DISCUSSION TITLE
is top.gg autovote going to be added?

### DISCUSSION THREAD

**Discussion by red78massive1573** [BridgeSenseMC/Dank-Memer-Grinder#116] [General]
the only autovote is see rn is discordbotlist

**Comment by saadndm** [comment]
I dont think theres a way for top.gg autovotes without using a browser and therefore solving their captchas

**Reply by ewesley2** [reply]
i don’t think i get any captchas if i use an adblocker. just occasionally disboard/top needs me to log back in, which is just one click.

**Comment by red78massive1573** [comment]
i think you can just use selenium and wait for user solve captchas. it could be a way as a reminder for user that they can vote

**Comment by saadndm** [comment]
Wont be added back for now
